FreeAPS/Sources/Models/IOBTick0.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+// "0" for oref0
+struct IOBTick0: JSON {
```

Comment:
this is a stripped down version of `IOBEntry` - that one has data we don't need and formats for dates are mixed (sometimes it's a string, sometimes it's a number of seconds - the JSON decoding was failing)